src/context/MapContext.tsx

```diff
@@ -12,6 +12,9 @@ import { Extent } from 'ol/extent';
 import BaseLayer from 'ol/layer/Base';
 import Layer from 'ol/layer/Layer';
 import Map from 'ol/Map';
+import Overlay from 'ol/Overlay';
+import { fromLonLat } from 'ol/proj';
+import STACLayer from 'ol-stac';
```

Comment:
Looks like we need to update the `package.json` if not done already to include `ol-stac`